pt/controller_1/config.py

```diff
@@ -16,4 +16,25 @@
 def get_config() -> dict:
         with open("config/config.yaml", "r") as file:
             data = yaml.safe_load(file)
-        return data
\ No newline at end of file
+        return data
+
+
+def get_pin_to_actuator_map() -> dict:
+    config = get_config()
+    actuator_map = {}
+    actuators = config.get("actuators", {})
+    for name, actuator in actuators.items():
+        actuator_map[actuator.get("pin")] = name
+    return actuator_map
+
+pin_to_relay_map = {
```

Comment:
These two dictionaries are to come from config. They need not be hard-coded in `config.py`.

Another oversight from me is the placement of config code. Most of the config parsing is actually taking place in `controller-1.py`. Please move those statements (fx, `sampling_period = config["plant"]["sensors"]["sampling_period"]`) into this module and write appropriate wrapper functions.